principles/fp-008-documented.md

```diff
@@ -45,6 +45,7 @@ Central to the issue of ontology documentation is ensuring transparency and trac
 
 - _Developer documentation:_
 
+- documentation about collaborating and submitting issues by creating a [Contributing.MD file](http://mozillascience.github.io/working-open-workshop/contributing/).
```

Comment:
Please uppercase the file part -> "CONTRIBUTING.md"
Also, might be good to point to the more detailed instructions here: http://obofoundry.org/principles/fp-020-responsiveness.html#implementation

Reply:
Done.